### ISSUE TITLE
Missing .rocks parser

### ISSUE THREAD

**Issue by swistaczek** [weppos/whois-parser#157]

**Comment by jarthod** [comment]
@swistaczekfor the record this project isn't really maintained by @weppos any more, so I ended up starting [my fork](https://github.com/jarthod/whois-parser) in which I merged some of the pending PRs and more fixes.

I did fix the `.rock` TLD parsing a while back in https://github.com/jarthod/whois-parser/commit/6354c4ead61afce8f0a92449ddef10dc3e2655ae **BUT** :unamused:  it looks like "Donuts Inc" rebranded to "Identity Digital Inc." in the meantime, so my fix no longer works and it'll require a small update.

Feel free to open a PR for this change on my fork, otherwise I'll try to fix it later.

This is how to use the fork:
```ruby
gem 'whois-parser', github: 'jarthod/whois-parser' # Unofficial but more up-to-date fork, check status at https://github.com/jarthod/whois-parser
```
